.dockerignore

```
@@ -1,3 +1,4 @@
 storage_pool
 **/*.iso
+**/minikube*
```

maybe change to 
```suggestion
minikube*
```
I wouldn't want any ``minikube*.py`` module we might want to add to be ignored

---

we're adding the entire minikube_home folder, so it should already be ignored (/home/assisted-test-infra/minikube_home/.minikube/machines/minikube/minikube.rawdisk)